packages/protocol/src/di/container-configuration.ts

```diff
@@ -52,8 +55,35 @@ export function resolveContainerConfiguration(...containerConfigurations: Contai
             if (config.add) {
                 distinctAdd(modules, ...asArray(config.add));
             }
+            if (config.replace) {
+                asArray(config.replace).forEach(replace => {
+                    const existingIndex = modules.findIndex(m => m instanceof FeatureModule && m.featureId === replace.featureId);
+                    if (existingIndex >= 0) {
+                        modules[existingIndex] = replace;
+                    } else {
+                        distinctAdd(modules, replace);
```

Comment:
I know you defined this as the intended behavior in the function comments but I feel that the `replace` behavior should still log a warning if it does not actually replace another module but defaults to the adding behavior. What do you think?